### PR TITLE
Attempts to fix problems with the battle royale gamemode

### PR DESCRIPTION
**Pull request by Moltijoe**
Workin on it
it works perfectly fine on my test server so i have no clue what's causing it

:cl:  
bugfix: attempts to make the battle royale gamemode not break 66% of the time
/:cl:

### CODE REVIEW

**Comment by MajManatee** [comment]
ah yes the only thing needed to fix it
adding elses

**Comment by Moja10** [comment]
If possible, could the crates be sent to a garbage collector after like 3 - 5 minutes of being open? They clog up the halls, especially when more spawn, making fighting really hard. Other than that, awesome that you are fixing the BR mode!

**Comment by Moltijoe** [comment]
> If possible, could the crates be sent to a garbage collector after like 3 - 5 minutes of being open? They clog up the halls, especially when more spawn, making fighting really hard. Other than that, awesome that you are fixing the BR mode!

i could probably do that, i'm just wanting to make sure it works first though.

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm [hunk 1]
```diff
@@ -64,9 +69,14 @@ GLOBAL_LIST_EMPTY(battleroyale_players) //reduce iteration cost
 	var/list/royalers = list()
 	if(GLOB.player_list.len <= 1) //It's a localhost testing
 		return
+	if(!GLOB.battleroyale_players.len) //sanity check for if this gets called before people are added to the list somehow
```
Comment (ToasterBiome) [inline yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: use length(x) proc

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: yogstation/code/game/gamemodes/battle_royale/battleroyale.dm (hunk 1 above)
Comment (Moltijoe) [reply yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: lazylen or length?

Comment (Moltijoe) [reply yogstation/code/game/gamemodes/battle_royale/battleroyale.dm]: also, should i change the other .len uses?

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**